codalab/worker/worker_run_state.py

```diff
@@ -251,13 +252,26 @@ def mount_dependency(dependency, shared_file_system):
 
         dependencies_ready = True
         status_messages = []
+        dependency_keys_to_paths: Dict[DependencyKey, str] = dict()
 
         if not self.shared_file_system:
             # No need to download dependencies if we're in the shared FS,
             # since they're already in our FS
             for dep in run_state.bundle.dependencies:
                 dep_key = DependencyKey(dep.parent_uuid, dep.parent_path)
-                dependency_state = self.dependency_manager.get(run_state.bundle.uuid, dep_key)
+
+                try:
+                    # Fetching dependencies from the Dependency Manager can fail.
+                    # Just update the download status on the next iteration of this transition function.
+                    dependency_state = self.dependency_manager.get(run_state.bundle.uuid, dep_key)
```

Comment:
Let's see if we could maybe revert this file to what's on master (so only the line `dependency_state = self.dependency_manager.get(run_state.bundle.uuid, dep_key)`). We might be able to do that if we use a default argument in the implementation of `get`.

Reply:
What would be a good default argument in this case?

Reply:
In line 40 of dependency_manager.py, change it to `dependencies, paths = self._fetch_state(default={'dependencies': {}, 'paths': set()})`?